src/routes/protect/estoque/product.js

```diff
@@ -8,5 +8,6 @@ router.get("/getEquipsByEntrance", productController.getEquipsByEntrance)
 router.get("/getAllNames", productController.getAllNames)
 router.get("/getProductByStockBase", productController.getProductByStockBase)
 router.get("/getAllVendas", productController.getAllVendas)
+router.get("/getByIdProducts", productController.getByIdProducts)
```

Comment:
dessa formar vamos conseguir pegar os dados via params no controller e domain
```suggestion
router.get("/getByIdProducts/:id", productController.getByIdProducts)
```